[2025-02-09T08:33:39.877+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-09T08:33:39.907+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: directory_and_file_verification_dag.verify_file_reliability manual__2025-02-09T08:33:30.548862+00:00 [queued]>
[2025-02-09T08:33:39.918+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: directory_and_file_verification_dag.verify_file_reliability manual__2025-02-09T08:33:30.548862+00:00 [queued]>
[2025-02-09T08:33:39.919+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-02-09T08:33:39.935+0000] {taskinstance.py:2889} INFO - Executing <Task(BranchPythonOperator): verify_file_reliability> on 2025-02-09 08:33:30.548862+00:00
[2025-02-09T08:33:39.949+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=294) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-09T08:33:39.953+0000] {standard_task_runner.py:72} INFO - Started process 296 to run task
[2025-02-09T08:33:39.960+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'directory_and_file_verification_dag', 'verify_file_reliability', 'manual__2025-02-09T08:33:30.548862+00:00', '--job-id', '1379', '--raw', '--subdir', 'DAGS_FOLDER/my_directory_and_file_checker.py', '--cfg-path', '/tmp/tmp7_ixlcug']
[2025-02-09T08:33:39.962+0000] {standard_task_runner.py:105} INFO - Job 1379: Subtask verify_file_reliability
[2025-02-09T08:33:40.015+0000] {task_command.py:467} INFO - Running <TaskInstance: directory_and_file_verification_dag.verify_file_reliability manual__2025-02-09T08:33:30.548862+00:00 [running]> on host 003b7fbea265
[2025-02-09T08:33:40.332+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='directory_and_file_verification_dag' AIRFLOW_CTX_TASK_ID='verify_file_reliability' AIRFLOW_CTX_EXECUTION_DATE='2025-02-09T08:33:30.548862+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-09T08:33:30.548862+00:00'
[2025-02-09T08:33:40.333+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-02-09T08:33:40.334+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-02-09T08:33:40.334+0000] {logging_mixin.py:190} INFO - Current task name:verify_file_reliability state:running start_date:2025-02-09 08:33:39.908235+00:00
[2025-02-09T08:33:40.335+0000] {logging_mixin.py:190} INFO - Dag name:directory_and_file_verification_dag and current dag run status:running
[2025-02-09T08:33:40.335+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-09T08:33:40.336+0000] {baseoperator.py:421} WARNING - BranchPythonOperator.execute cannot be called outside TaskInstance!
[2025-02-09T08:33:40.337+0000] {logging_mixin.py:190} INFO - camarche22 /opt/***/files ['.csv'] ['utf-8', 'ISO-8859-1', 'Windows-1252']
[2025-02-09T08:33:40.346+0000] {my_directory_and_file_checker.py:111} ERROR - ‚ùå Erreur d'encodage (utf-8) pour √âcritures.csv: 'utf-8' codec can't decode byte 0xb0 in position 1: invalid start byte
[2025-02-09T08:33:40.368+0000] {logging_mixin.py:190} INFO - 
üìÇ Contenu du fichier (ISO-8859-1): √âcritures.csv
[2025-02-09T08:33:40.376+0000] {logging_mixin.py:190} INFO -   Date comptabilisation       N¬∞ lot Type √©criture
0            31/12/2024  A-BNDK 2304    Production
1            31/12/2024  B-BNDK 2304    Production
2            31/12/2024  C-BNDK 2304    Production
3            31/12/2024  D-BNDK 2304    Production
4            31/12/2024  E-BNDK 2304    Production
[2025-02-09T08:34:47.133+0000] {logging_mixin.py:190} INFO - ‚úÖLignes  ('‚úÖ Le fichier respecte les colonnes attendues et les types de donn√©es.', True) 9854
[2025-02-09T08:34:47.136+0000] {logging_mixin.py:190} INFO - merci
[2025-02-09T08:34:47.150+0000] {base.py:84} INFO - Retrieving connection 'mysql_conn'
[2025-02-09T08:34:47.168+0000] {logging_mixin.py:190} INFO - (('attendancedb',), ('caprostockdb',), ('information_schema',), ('mysql',), ('netvision_db',), ('performance_schema',), ('pointing_db',), ('posgpt_db',), ('pythondb',), ('sys',), ('testimportdb',))
[2025-02-09T08:34:47.223+0000] {logging_mixin.py:190} INFO - ‚ö†Ô∏è Donn√©es supprim√©es avant insertion.
[2025-02-09T08:34:48.418+0000] {sql.py:691} INFO - Loaded 1000 rows into temp_ecc so far
[2025-02-09T08:34:49.437+0000] {sql.py:691} INFO - Loaded 2000 rows into temp_ecc so far
[2025-02-09T08:34:50.201+0000] {sql.py:691} INFO - Loaded 3000 rows into temp_ecc so far
[2025-02-09T08:34:50.992+0000] {sql.py:691} INFO - Loaded 4000 rows into temp_ecc so far
[2025-02-09T08:34:51.802+0000] {sql.py:691} INFO - Loaded 5000 rows into temp_ecc so far
[2025-02-09T08:34:52.540+0000] {sql.py:691} INFO - Loaded 6000 rows into temp_ecc so far
[2025-02-09T08:34:53.364+0000] {sql.py:691} INFO - Loaded 7000 rows into temp_ecc so far
[2025-02-09T08:34:54.084+0000] {sql.py:691} INFO - Loaded 8000 rows into temp_ecc so far
[2025-02-09T08:34:54.851+0000] {sql.py:691} INFO - Loaded 9000 rows into temp_ecc so far
[2025-02-09T08:34:55.633+0000] {sql.py:694} INFO - Done loading. Loaded a total of 9854 rows into temp_ecc
[2025-02-09T08:34:55.635+0000] {logging_mixin.py:190} INFO - connexion sql reussie True
[2025-02-09T08:34:55.644+0000] {logging_mixin.py:190} INFO - Afficher   Accounting_Date   Lot_Number Type_ecriture
0      2024-12-31  A-BNDK 2304    Production
1      2024-12-31  B-BNDK 2304    Production
2      2024-12-31  C-BNDK 2304    Production
3      2024-12-31  D-BNDK 2304    Production
4      2024-12-31  E-BNDK 2304    Production
[2025-02-09T08:34:55.645+0000] {python.py:240} INFO - Done. Returned value was: end
[2025-02-09T08:34:55.645+0000] {branch.py:38} INFO - Branch into end
[2025-02-09T08:34:55.646+0000] {skipmixin.py:230} INFO - Following branch ('end',)
[2025-02-09T08:34:55.659+0000] {skipmixin.py:278} INFO - Skipping tasks []
[2025-02-09T08:34:55.692+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-09T08:34:55.694+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=directory_and_file_verification_dag, task_id=verify_file_reliability, run_id=manual__2025-02-09T08:33:30.548862+00:00, execution_date=20250209T083330, start_date=20250209T083339, end_date=20250209T083455
[2025-02-09T08:34:55.706+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-02-09T08:34:55.707+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-02-09T08:34:55.707+0000] {logging_mixin.py:190} INFO - Dag name:directory_and_file_verification_dag queued_at:2025-02-09 08:33:30.583876+00:00
[2025-02-09T08:34:55.708+0000] {logging_mixin.py:190} INFO - Task hostname:003b7fbea265 operator:BranchPythonOperator
[2025-02-09T08:34:55.730+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-09T08:34:55.757+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-09T08:34:55.761+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
